changelog/unreleased/issue-5254.toml

```
@@ -0,0 +1,4 @@
+type="a"
+message="Display System logs from Kafka topic using file beats"
```

We should specify that this is a new Graylog Input:
```suggestion
message="Added new Beats Kafka input to ingest system logs from Kafka topic using Filebeat."
```